### DISCUSSION TITLE
bundle react with C#+sqlite as client side db

### DISCUSSION THREAD

**Discussion by abhijitpatil181171** [tauri-apps/tauri#8485] [General]
how Configure Tauri to bundle both the React frontend and the C# backend (with sqlite as client side db )together into a single executable file.
can you provide some documentation to do so

**Comment by amrbashir** [comment]
Frontend are bundled by pointing [`distDir`](https://tauri.app/v1/api/config#buildconfig.distdir) to the directory that contains your frontend files (whether be it react or anything else) and for C# backend, you can embed it as an external binary, see https://tauri.app/v1/guides/building/sidecar and https://tauri.app/v1/guides/building/resources
